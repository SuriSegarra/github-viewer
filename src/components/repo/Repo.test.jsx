import React from 'react';
import { shallow } from 'enzyme';
import Repo from './Repo';

describe('Repo', () => {
  it('matches a repo snapshot', () => {
    const wrapper = shallow(<Repo 
      name='Surelis Segarra' 
      html_url='https://github.com/SuriSegarra'
    />);
    expect(wrapper).toMatchSnapshot();
  });
});
