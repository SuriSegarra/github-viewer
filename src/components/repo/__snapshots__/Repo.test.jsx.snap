// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repo matches a repo snapshot 1`] = `
<li>
  <p>
    Surelis Segarra
  </p>
  <p>
    https://github.com/SuriSegarra
  </p>
</li>
`;
